app/assets/stylesheets/application.css.scss

```diff
@@ -14,3 +14,15 @@
  *= require_self
  *= require bootstrap.min
  */
+
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 60px;
+  background-color: #F5F5F5;
```

Comment:
Color literals like `#F5F5F5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F5F5F5` should be written as `#f5f5f5`